### PR TITLE
Issue 2 user can save search results - Nick Sturz

### PR DESCRIPTION
**Pull request by nsturz**
Video is in slack!

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -29,6 +29,16 @@ a {
   margin-top: 1%;
 }
 
+#back-to-search-button-footer {
```
Comment (TimelikeClosure) [inline css/styles.css]: Please avoid using ids as CSS style targets.

File: css/styles.css [hunk 2]
```diff
@@ -29,6 +29,16 @@ a {
...
+  width: 300px;
+  height: 50px;
+  font-size: 35px;
```
Comment (TimelikeClosure) [inline css/styles.css]: Try to use recommended unit for styling properties: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 1 above)
Comment (nsturz) [reply css/styles.css]: fixed!

File: css/styles.css (hunk 2 above)
Comment (nsturz) [reply css/styles.css]: fixed!

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**